adapter/config/default_config.go

```diff
@@ -134,6 +139,7 @@ var defaultConfig = &Config{
 			AllowPartialMessage: false,
 			PackAsBytes:         false,
 		},
+		EnableMtlsApis: false,
```

Comment:
Shall we change this to MTLSAPIsEnabled?

Reply:
Changed